src/API/Management/GenericResource.php

```diff
@@ -75,17 +75,15 @@ protected function normalizePagination(array $params, $page = null, $per_page =
      *
      * @return array
      */
-    protected function normalizeIncludeTotals(array $params, $include_totals = null)
+    protected function normalizeIncludeTotals(array $params, $include_totals = false)
     {
         // User parameter include_totals if params does not have the key.
         if (! isset( $params['include_totals'] )) {
             $params['include_totals'] = $include_totals;
         }
 
-        // If include_totals is set (not null), then make sure we have a boolean.
-        if (isset( $params['include_totals'] )) {
-            $params['include_totals'] = boolval( $params['include_totals'] );
-        }
+        // Make sure we have a boolean.
+        $params['include_totals'] = filter_var( $params['include_totals'], FILTER_VALIDATE_BOOLEAN );
```

Comment:
We're still converting `null` to `false` here. Suggest:

```suggestion
        $params['include_totals'] = filter_var( $params['include_totals'], FILTER_VALIDATE_BOOLEAN, FILTER_NULL_ON_FAILURE );
```

Reply:
Done